File: AmpedBiz/AmpedBiz.Service/Dto/UnitOfMeasureClass.cs
using System.Collections.Generic;

namespace AmpedBiz.Service.Dto
{
    public class UnitOfMeasureClass
    {
        public virtual string Id { get; set; }

        public virtual string Name { get; set; }

        public virtual List<UnitOfMeasure> Units { get; set; }
    }

    public class UnitOfMeasureClassPageItem
    {
        public virtual string Id { get; set; }

        public virtual string Name { get; set; }
    }
}

File: AmpedBiz/AmpedBiz.Web/src/common/models/unit-of-measure-class.ts
import {UnitOfMeasure} from './unit-of-measure';

export interface UnitOfMeasureClass {
  id?: string;
  name?: string;
  units?: UnitOfMeasure[];
}

export interface UnitOfMeasureClassPageItem {
  id?: string;
  name?: string;
}